src/style.css

```diff
@@ -0,0 +1,121 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+
+  --blue-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.product-card {
+  display: flex;
+  justify-content: center;
+  width: min-content;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__content-container {
+  padding: 30px 15px 15px;
+  width: 200px;
+}
+
+.product-card__image-container {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 45px;
+}
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__text {
+  margin-bottom: 6px;
+  font-weight: 500;
+  font-size: 12px;
+  background-color: #fff;
+  line-height: 15px;
+}
+
+.product-card__product-code {
+  font-weight: 400;
+  font-size: 11px;
+  margin-bottom: 15px;
+}
+
+.product-card__rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+.stars {
+  display: inline-flex;
+  justify-content: space-between;
+}
+
+.stars__star {
+  width: 15px;
+  height: 15px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--0 :nth-child(n),
+.stars--1 :nth-child(n + 2),
+.stars--2 :nth-child(n + 3),
+.stars--3 :nth-child(n + 4),
+.stars--4 :nth-child(n + 5) {
+  background-image: url(images/star.svg);
+}
```

Comment:
I think something is wrong here. You have no stars--0, stars--1, stars--2, stars--3 classes.